src/main/java/com/faforever/client/chat/ChannelTabController.java

```diff
@@ -273,6 +273,10 @@ protected WebView getMessagesWebView() {
 
   @Override
   protected void onMention(ChatMessage chatMessage) {
+    if (preferencesService.getPreferences().getNotification().isNotifyOnAtMentionEnabled()
```

Comment:
Ok I hear NotifyOnAtMention and I thing u are either notified if somebody mentions you with an @ sign or you are not at all. Maybe name it NotifyOnAtMentioningOnly. So the purpose gets clear. In Intelij you can refactor name by selecting the name and pressing (Umschalten+F6)